core/commands/pom.xml

```diff
@@ -43,5 +43,17 @@
       <groupId>org.apache.karaf.shell</groupId>
       <artifactId>org.apache.karaf.shell.core</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.opennms</groupId>
+      <artifactId>opennms-dao-api</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.opennms</groupId>
+      <artifactId>opennms-model</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.opennms</groupId>
+      <artifactId>opennms-web-api</artifactId>
```

Comment:
This dependency might be problematic: https://github.com/OpenNMS/opennms/blob/develop/opennms-web-api/pom.xml

Reply:
I thought that was left over from an earlier revision, but `ShowRequisitions` fails to build when I remove it from the pom:
```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.11.0:compile (default-compile) on project org.opennms.core.commands: Compilation failure: Compilation failure:
[ERROR] /home/dino2gnt/src/opennms/core/commands/src/main/java/org/opennms/core/commands/ShowRequisitions.java:[37,44] package org.opennms.netmgt.provision.persist does not exist
[ERROR] /home/dino2gnt/src/opennms/core/commands/src/main/java/org/opennms/core/commands/ShowRequisitions.java:[38,56] package org.opennms.netmgt.provision.persist.requisition does not exist
[ERROR] /home/dino2gnt/src/opennms/core/commands/src/main/java/org/opennms/core/commands/ShowRequisitions.java:[46,13] cannot find symbol
[ERROR]   symbol:   class ForeignSourceRepository
[ERROR]   location: class org.opennms.core.commands.ShowRequisitions
[ERROR] /home/dino2gnt/src/opennms/core/commands/src/main/java/org/opennms/core/commands/ShowRequisitions.java:[51,23] cannot find symbol
[ERROR]   symbol:   class Requisition
[ERROR]   location: class org.opennms.core.commands.ShowRequisitions
[ERROR] /home/dino2gnt/src/opennms/core/commands/src/main/java/org/opennms/core/commands/ShowRequisitions.java:[57,18] cannot find symbol
[ERROR]   symbol:   class Requisition
[ERROR]   location: class org.opennms.core.commands.ShowRequisitions
```

I'm happy to move these commands somewhere that the dependencies have less impact, but I don't know where that would be. Perhaps there should be a larger conversation about where utility commands such as these should live, because I have a laundry list of commands I'd like to add as my time permits.

Reply:
Ah, I see what's going on.  `opennms-web-api` provides interfaces to the same classes as `opennms-provision-persistence` so puling that in makes those bits available, but all I really need to include is `opennms-provision-persistence` and bypass the intermediary api package.